src/main/resources/add-ons-to-index.json

```diff
@@ -426,7 +441,7 @@
       "type": "OMOD",
       "name": "chirdlutilbackports",
       "tags": [
-        "chica"
+        "chica", "Chirdlutil", "Backsports"
```

Comment:
Taking part of the module name and including it as a tag isn't really helpful, kindly try to add meaningful tags.